src/player.cpp

```diff
@@ -9587,10 +9587,11 @@ void player::do_read( item &book )
                                           pgettext( "memorial_female", "Reached skill level %1$d in %2$s." ),
                                           skill_level.level(), skill->name() );
                     }
-                    lua_callback( "on_skill_increased" );
+                    // possible callback arguments: skill_increase_type, skill_id, skill_level_new
+                    lua_callback( "on_skill_increased", "book", skill.c_str(), to_string( originalSkillLevel + 1 ).c_str() );
                 } else {
                     add_msg( m_good, _( "%s increases their %s level." ), learner->disp_name().c_str(),
-                             skill.obj().name().c_str() );
+                             skill.obj().name().c_str(), skill.obj().name().c_str() );
```

Comment:
The message foprmat string only contains two "%s", and there are already two arguments for thos placeholders. Why have you added a third argument (identicaly to the second one), that is never used?

Reply:
I do not remember why I duplicated this :)